### PR TITLE
General content of the page

### PR DESCRIPTION
**Pull request by JbirdL86**
By mistake user-column branch was merged to the master before making this pull request.

### CODE REVIEW

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: style.css [hunk 1]
```diff
@@ -1,545 +1,6 @@
...
-    }
-	
-    #menu{
-	  background-color: transparent;
-	  border: none;
-	  height: 29px;
-	  cursor: pointer;
-	}
-	
-	#menucont{
```
Comment (andresporras3423) [inline style.css:36]: Don't overuse ID selectors

File: style.css [hunk 2]
```diff
@@ -1,545 +1,6 @@
...
-	}
-
-	/*Making the info division icons look gray*/
-	#info i{
-	  color: gray;
-	}
-
-	/*Giving a more specific gray to the add list icon in the info division*/
-	.add-list-color{
-		color: #d1cccc !important;
```
Comment (andresporras3423) [inline style.css:245]: Don't use !important

File: style.css [hunk 3]
```diff
@@ -1,545 +1,6 @@
...
-	  position: relative;
-	  top: 4px;
-	  display: flex;
-	  align-items: center;
-	  justify-content: space-between;
-	}
-
-	#icons-right div{
-	  cursor: pointer;
-	}
```
Comment (andresporras3423) [inline style.css:202]: This selector and selector on line 120 both have the same style definition. Don't repeat yourself, group selectors.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @JbirdL86 @Huemac-Alfredo** 
Good job with your first submission for this project :muscle:  
Your HTML pass the validations :clap: :clap: :clap:  
There are some issues to solve before you can go ahead:
- [ ] for your next "review request" please provide a more meaningful description
- [ ] Your live demo is not working, you can use  https://raw.githack.com/ to create a live demo for this project.
- [ ] Don't use important
- [ ] Don't overuse ID Selectors
- [ ] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

File: style.css (hunk 3 above)
Comment (JbirdL86) [reply style.css:202]: Done

File: style.css (hunk 2 above)
Comment (JbirdL86) [reply style.css:245]: Done

File: style.css (hunk 1 above)
Comment (JbirdL86) [reply style.css:36]: I'm not sure how to solve this

**Comment by JbirdL86** [comment]
@andresporras3423 I'm not sure if this pull request needs to stay or we need to create a new one?

**Comment by elmejdki** [comment]
# Hi, @andresporras3423 create a new pull request and remember to never close a pull request that was not approved.
## Happy Coding :tophat:

**Comment by elmejdki** [comment]
# @andresporras3423 Actually, I think you need to create a new repository please contact me on slack so I can explain to you the process. you will find me as _Zakariae El Mejdki_.

**Pull request closed without merging**